main.go

```diff
@@ -405,6 +407,14 @@ func run(state overseer.State) {
 	if *jobReportFile != nil {
 		jobReportWriter = *jobReportFile
 	}
+
+	// Parse --results flag.
+	if *onlyVerified {
+		r := "verified"
+		results = &r
+	}
+	parsedResults, err := parseResults(results)
```

Comment:
Does this error just get thrown away? If so, is that what we want?

Reply:
The dangers of distracted coding. 😬

Reply:
Fixed.
```sh
$ ./trufflehog git file:///tmp/trufflehog --results=8hoursofsleep
🐷🔑🐷  TruffleHog. Unearth your secrets. 🐷🔑🐷

2024-03-11T13:23:26-04:00       error   trufflehog      failed to configure results flag        {"error": "invalid value '8hoursofsleep', valid values are 'verified,unknown,unverified'"}
```